fpage/views.py

```diff
@@ -66,13 +66,51 @@ def logoutUser(request):
 	logout(request)
 	return redirect('fpage:home')
 
+@login_required(login_url='fpage:login')
+def profileView(request):
+    all_info=UserProfile.objects.filter(user=request.user)
+    context={
+        'all_info':all_info,
+    }
+    return render(request,'fpage/profile.html',context)
+
+@login_required(login_url='fpage:login') 
+def profileUpdate(request): 
+    profile= get_object_or_404(UserProfile, user=request.user)
+    form = ProfileForm(initial={'firstname': profile.first_name , 'lastname': profile.last_name, 'email': profile.email, 'phone_num': profile.phone_num, 'fav_genre':profile.fav_genre , 'profile_image':profile.profile_image})
+    if request.method == "POST":  
+        form = ProfileForm(request.POST, instance=profile)  
+        if form.is_valid():  
+            try: 
+                form.save() 
+                model = form.instance
+                return redirect('fpage:profile')
+            except Exception as e: 
```

Comment:
Instead of passing the exception  try to show error messages

Reply:
@ExpressHermes Like this?
![image](https://user-images.githubusercontent.com/45357209/98633372-46a80a00-2347-11eb-83ae-54e269ceb282.png)

Reply:
Also provide proper errors `form.errors` to send the message for why the updation/creation failed

Reply:
Also try to send these in normal PR comments as viewing them here is a inconvenient

Reply:
yes i by mistake did it here